src/components/inventoryList/inventoryList.js

```diff
@@ -182,9 +207,13 @@ class InventoryList extends React.Component {
       );
     }
 
-    const updatedPerPage = query?.[RHSM_API_QUERY_TYPES.LIMIT] || perPageDefault;
-    const updatedPage = query[RHSM_API_QUERY_TYPES.OFFSET] / query[RHSM_API_QUERY_TYPES.LIMIT] + 1 || 1;
-    const isLastPage = updatedPage === Math.ceil(itemCount / updatedPerPage);
+    const updatedPerPage = query[RHSM_API_QUERY_TYPES.LIMIT] || perPageDefault;
+    const updatedOffset = query[RHSM_API_QUERY_TYPES.OFFSET];
+    const isLastPage =
```

Comment:
optional: since we're already offsetting basic paging calcs in helpers, maybe move "isLastPage" into the paginationHelpers too